types/pdfmake/interfaces.d.ts

```diff
@@ -1310,9 +1310,11 @@ export interface ContentQr extends ContentBase, ForbidOtherElementProperties<"qr
      * Size in `pt` to fit the QR code in.
      *
      * As QR codes are square, this essentially sets both the width and height.
+     * pdfmake seems to shrink QR codes using certain fixed factors, so it might actually be smaller
+     * than the value given here.
      *
      * Defaults to a heuristic depending on the pixel size of the QR code (see {@link version})
-     * so that every pixel is `5pt` large.
+     * so that every pixel is about `4.5pt` large.
```

Comment:
According to the implementation it's 5px [the implementation uses 5px](https://github.com/bpampuch/pdfmake/blob/master/src/qrEnc.js#L755).

Reply:
Huh. You're right, I think I got confused when adding the padding in https://github.com/kryops/pdfmake-html-renderer/commit/1460bb46f9dc9619afe2cd93d79cdd550c18f228 that my QR code was wider than pdfmake's - but that is because I was comparing QR codes with a different amount of QR pixels.

Reverted by removing the commit.